README.md

```diff
@@ -1,134 +1,329 @@
-# skuid
-
-`skuid` is a command line application for interacting with Skuid metadata, on both Salesforce Platform and Skuid Platform.
-It provides ways to deploy and retrieve Skuid application metadata to and from either a Salesforce Platform org running the Skuid managed package or a Skuid Platform Site.
-
-## Motiviation
+# skuid CLI
```

Comment:
Personally, I'd rather this stay `skuid` for consistency.

Reply:
That makes sense. On the docs site I think we may still got with `skuid CLI` (or any other suggestions?) to better separate it in our table of contents. Just don't want one tiny topic to be confused with Skuid as a whole.